### DISCUSSION TITLE
Search engine indexing

### DISCUSSION THREAD

**Discussion by emveepee** [OpenSageTV/Community#24] [GitHub Discussions usage and questions]
Is this site searchable via the normal search engines?  I tried searching many things like 

```
narflex site:https://github.com/OpenSageTV/Community/discussions/.
```
and I don't get any results.  By far my most important use of the old forums wa the wealth of knowledge available.  Granted I don't expect that much expert information from this forum anymore using github searching isn't going to really work for me.

**Comment by wnjj** [comment]
The built-in search at the top works fine on GitHub.  You just need to click on the “discussions” filter after searching.  As you said though, it may be awhile if ever before there is a bunch of useful stuff to find.  At least it should all be a bit more relevant. 

https://github.com/search?q=repo%3AOpenSageTV%2FCommunity+Narflex&type=discussions

**Reply by emveepee** [reply]
I was using my example to show this discussion group wasn't indexed.  When you want to search the site, it will work but I prefer DDG or Google which is really were the value of the old forum comes in.  A lot of SageTV info is good general knowledge for PVR.

**Reply by wnjj** [reply]
I guess I'm not understanding why using DDG or Google is any better than native GitHub searches for the newer info.  For me I agree that it was definitely easier to find stuff using outside search than vBulletin's search function on the old forum but I suspect GitHub search will work fine.

Since the old forum will still be DDG/Google searchable with "site:sagetv.com" you can still find all of the old stuff going forward.  What's unavoidable is that you will also have to search on GitHub to see if there is any updated info on the topic.
